src/backend/web/routes/feeds.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+const Feed = require('../../data/feed');
+const { getFeeds, getFeedsCount } = require('../../utils/storage');
+const { logger } = require('../../utils/logger');
+
+const feeds = express.Router();
+
+feeds.get('/', async (req, res) => {
+  let ids;
+  let feedsCount;
+
+  try {
+    ids = await getFeeds();
+    feedsCount = await getFeedsCount();
```

Comment:
I think you should either parallelize these calls:

`[ids, feedsCount] = await Promise.all([getFeeds(), getFeedsCount()]);`

Or just use `ids.length`.